Code/GraphMol/SmilesParse/SmilesWrite.cpp

```diff
@@ -37,6 +38,53 @@ bool inOrganicSubset(int atomicNumber) {
   return atomicSmiles[idx] == atomicNumber;
 }
 
+namespace {
+std::string getAtomChiralityInfo(const Atom *atom) {
+  auto allowNontet = Chirality::getAllowNontetrahedralChirality();
```

Comment:
The first "t" in `allowNontet` should probably be uppercase

Reply:
I thought that, but it would be inconsistent with getAllowNontetrahedral…

Reply:
oh, ok